src/realm.js

```diff
@@ -614,4 +621,25 @@ export class Realm {
     }
     for (; i < generatorBody.length; i++) realmGeneratorBody.push(generatorBody[i]);
   }
+
+  handleError(error: IntrospectionThrowCompletion): boolean {
```

Comment:
I thought we wanted to limit it to that for now, no? It's true that we may want to handle other errors in the future, but would be hard to write generic code to convert them to native. We can adjust this handler as needed in the future, since it's not used by the external client (that is abstracted in the NativeError).